packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -294,9 +319,238 @@ const MultiCodeLayout = ({
+  object-fit: cover;
+`;
+
+const ImgContainer = styled(FlexBox)`
+  overflow: hidden;
+`;
+
+const Image = ({
+  src,
+  alt = '',
+  imgContainerProps,
+  imgProps
+}: {
+  src: string;
+  alt: string;
+  imgContainerProps?: ComponentProps<typeof FlexBox>;
+  imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+}) => (
+  <ImgContainer data-testid="ImgContainer" {...imgContainerProps}>
+    <Img data-testid="Img" src={src} alt={alt} {...imgProps} />
+  </ImgContainer>
+);
+
+/**
+ * Horizontal Image layout with optional Title and Description
+ */
+const HorizontalImage = ({
+  src,
+  alt,
+  title,
+  titleProps,
+  description,
+  descriptionProps,
+  imgProps,
+  imgContainerProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  src: string;
+  alt: string;
+  title?: string | ReactNode;
+  titleProps?: ComponentProps<typeof Text>;
+  description?: string | ReactNode;
+  descriptionProps?: ComponentProps<typeof Text>;
+  imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+  imgContainerProps?: ComponentProps<typeof FlexBox>;
+}) => {
+  return (
+    <Slide {...rest}>
+      <Image
+        src={src}
+        alt={alt}
+        imgContainerProps={{
+          width: '100%',
+          ...imgContainerProps
+        }}
+        imgProps={imgProps}
+      />
+      {title ? (
+        <Heading textAlign="left" margin="0 0" {...titleProps}>
+          {title}
+        </Heading>
+      ) : null}
+      {description ? (
+        <Text margin="0 0" {...descriptionProps}>
+          {description}
+        </Text>
+      ) : null}
+    </Slide>
+  );
+};
+
+/**
+ * Image and List layout with optional Title
+ */
+const VerticalImage = ({
+  src,
+  alt,
+  title,
+  titleProps,
+  listType = 'unordered',
+  listItems,
+  animateListItems = false,
+  listProps,
+  imgProps,
+  imgContainerProps,
+  position = 'left',
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  src: string;
+  alt: string;
+  listItems: ReactNode[];
+  title?: string | ReactNode;
+  titleProps?: ComponentProps<typeof Heading>;
+  listType?: 'unordered' | 'ordered';
+  animateListItems?: boolean;
+  listProps?: ComponentProps<typeof UnorderedList & typeof OrderedList>;
+  imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+  imgContainerProps?: ComponentProps<typeof FlexBox>;
+  position?: 'right' | 'left';
+}) => {
+  return (
+    <Slide {...rest}>
+      {title ? <Heading {...titleProps}>{title}</Heading> : null}
+      <Grid
+        gridColumnGap={2}
+        gridTemplateColumns={'repeat(2, 1fr)'}
+        height={'100%'}
+      >
+        <FlexBox justifyContent="start">
+          {/* @ts-ignore TODO: Resolve this in follow-up */}
+          <Ul
+            items={listItems}
+            animate={animateListItems}
+            type={listType}
+            {...listProps}
+          />
+        </FlexBox>
+        <Image
+          src={src}
+          alt={alt}
+          imgContainerProps={{
+            order: position === 'right' ? 1 : -1,
```

Comment:
Normally I would agree you. For this use case I think it is fine and maybe even preferable to use `order` for toggling the position here. Specifically, this usage is for ordering an image that corresponds to a list. My understanding of screenreader-friendly image placement is that images should follow descriptive content, like titles and lists, so that as a person tabbing through the markup the image is encountered near the end/last. `order` doesn't change the order of the markup, just the placement, so I think this achieves both in an accessible way.

Always happy to learn more about accessibility so if there's more you would add, please do.

Reply:
I see your point. I think there are (rare?) cases where the image alt text contains something vital to the understanding of the rest of the slide, but there are just as many or more cases where the opposite is the case, so your approach is good.